resources/geocoding/en/93.txt

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2012 The Libphonenumber Authors 
+
+# Licensed under the Apache License, Version 2.0 (the "License"); 
+# you may not use this file except in compliance with the License. 
+# You may obtain a copy of the License at 
+ 
+# http://www.apache.org/licenses/LICENSE-2.0 
+ 
+ 
+# Unless required by applicable law or agreed to in writing, software 
+# distributed under the License is distributed on an "AS IS" BASIS, 
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+# See the License for the specific language governing permissions and 
+# limitations under the License. 
+# Generated from: 
+# https://www.itu.int/dms_pub/itu-t/oth/02/02/T02020000010002PDFE.pdf [2008-08-28] 
```

Comment:
2015-01-13 is for Mobile networks, not for fixed lines.
Please look at Page 2(It's for fixed lines) you will see 2008-Viii-28 which means 2008-08-28.
Geocoding is based on fixed lines, not mobile networks!
What is your opinion?